docker/cli
# add docker cp completion for container ids

**- What I did**
Modified the fish shell completion for `docker cp` such that it completes running container IDs as well as local file paths.

**- How I did it**
Added a new line to the competion file for fish, and edited a couple of the other `cp` completions to allow file paths.

**- How to verify it**
1) Run a docker container
2) Open fish shell
3) Type `docker cp ` followed by TAB to initiate completion
4) Verify that file paths are completed, as well as the container ID of any running containers

**- Description for the changelog**
Update fish shell completion for `docker cp` to complete container IDs as well as file paths.


**- A picture of a cute animal (not mandatory but encouraged)**
![Image of Hamster](https://www.tenerifenews.com/wp-content/uploads/2018/06/PAGE-20-SYRIAN-HAMSTER.jpg)
